res/md/common-intro.md

```diff
@@ -46,10 +46,10 @@ The main aim of a Content Information Type Specification (CITS) is to:
 
 - Define, in technical terms, how data and metadata must be formatted and placed within a CSIP Information Package to achieve interoperability in exchanging specific Content Information.
 
-The number of possible Content Information Type Specifications is unlimited. For a list of existing Content Information Type Specifications see the relevant section in the Common Specification for Information Packages.
+The number of possible Content Information Type Specifications is unlimited. For a list of existing Content Information Type Specifications see the DILCIS Board webpage.
```

Comment:
"DILCIS Board webpage" Should this be a link?

Reply:
Have added it, its further down in the document but for clarity.